back-nest/src/sсrapers/scraper-rozetka.service.ts

```diff
@@ -46,12 +46,12 @@ export class ScraperRozetkaService {
         $(element).find('.goods-tile__picture img').attr('data-src');
       const source = 'ROZETKA';
 
-      if (title && newPrice && profileImage) {
-        const productData: Omit<ProductType, 'id'> = {
+      if (title && price && profileImage) {
+        const productData: ScrapedProduct = {
           title,
           subtitle,
           description,
-          newPrice,
+          price,
           specifications,
           type,
           profileImage,
```

Comment:
Keep `newPrice: null` field for structure consistency

```suggestion
          profileImage,
          newPrice: null,
```